runtime/aws/AWS Backup/check-backup-recovery-point-encryption.ts

```diff
@@ -0,0 +1,138 @@
+import {
+	BackupClient,
+	ListBackupVaultsCommand,
+	ListRecoveryPointsByBackupVaultCommand,
+	DescribeRecoveryPointCommand
+} from "@aws-sdk/client-backup";
+import { printSummary, generateSummary } from "../../utils/string-utils.js";
+import { ComplianceStatus, type ComplianceReport, type RuntimeTest } from "../../types.js";
+
+async function checkBackupRecoveryPointEncryption(
+	region: string = "us-east-1"
+): Promise<ComplianceReport> {
+	const client = new BackupClient({ region });
+	const results: ComplianceReport = {
+		checks: []
+	};
+
+	try {
+		// Get all backup vaults first
+		const vaults = await client.send(new ListBackupVaultsCommand({}));
+
```

Comment:
use nextToken pagination to get all, in helper

Reply:
done